### ISSUE TITLE
Cannot use cards in homepage widgets

### ISSUE THREAD

**Issue by anhallbe** [infor-design/enterprise-ng#330]
**Describe the bug**
When widget has a `.card` inside of it, for instance when trying to add a header to a `soho-listview`, the card is treated as a widget in the layout.

**To Reproduce**
Given a homepages with 2 widgets, where each widget has one card (a list with a header) in it:
```html
<div soho-homepage>
  <div soho-widget>
    <div soho-widget-header>
      <div soho-widget-title>
        <h2>First Widget Title</h2>
      </div>
    </div>
    <div soho-widget-content>
      <h1>BEGIN First Widget Content</h1>
      <div class="card">
        <div class="card-header">
          <h2 class="card-title">First List Title</h2>
        </div>
        <div class="card-content">
          <soho-listview>
            <li soho-listview-item *ngFor="let item of list">
              {{item}}
            </li>
          </soho-listview>
        </div>
      </div>
      <h1>END First Widget Content</h1>
    </div>
  </div>

  <div soho-widget>
    <div soho-widget-header>
      <div soho-widget-title>
        <h2>Second Widget Title</h2>
      </div>
    </div>
    <div soho-widget-content>
      <h1>BEGIN Second Widget Content</h1>
      <div class="card">
        <div class="card-header">
          <h2 class="card-title">Second List Title</h2>
        </div>
        <div class="card-content">
          <soho-listview>
            <li soho-listview-item *ngFor="let item of list">
              {{item}}
            </li>
          </soho-listview>
        </div>
      </div>
      <h1>END Second Widget Content</h1>
    </div>
  </div>
</div>
```
The template above will create 4 incorrectly placed widgets.

**Expected behavior**
The given template should produce 2 widgets next to each other, and the lists should be contained inside the widgets.

**Version**
- ids-enterprise-ng: 5.0.0 (The problem has been around since at least August, version 4. )

**Screenshots**
![image](https://user-images.githubusercontent.com/4991954/51470438-2b37cd80-1d74-11e9-8f96-430005825ba1.png)

**Platform**
All

**Additional context**
https://jira.infor.com/browse/LIME-3772
Problem is possibly in `ids-enterprise`, since it can be reproduced without the angular directives.
Repro code: https://github.com/anhallbe/ids-repro/blob/card-in-widget/src/app/app.component.html

**Comment by tmcconechy** [comment]
Why would you need to do this? From what i can see the card would never be needed. The widget itself IS a card. (card is the same as widget). So if you did something like this you would have a white background in the widget-content section. So you would just put the list directly in there.

I think be adding the extra card the algorithm is thinking its another box. So does it work if you just take out the card?


```
<div class="widget">
   <div class="widget-header">
      <h2 class="widget-title">Dom Order 1, 1x1, #Small Widget</h2>
     </div>
     <div class="widget-content"> 
     <--List Here-->
    </div>
</div>
```

**Comment by anhallbe** [comment]
Most of the cases I've seen is when Homepages Widget developers want one or more lists inside a widget, and they want to add a header to the list(s).

It's definitely possible to do without the extra `.card`, there's just no IDS sample demonstrating how to do it (in a consistent way), as far as I know.

If this is outside the scope of IDS, I guess we could make something of our own

**Comment by anhallbe** [comment]
All right!
Sorry, I must've missed that. It's exactly what I was looking for. Thanks!

I still think it's a bit strange that a card inside a widget can mess up the layout of the whole page, but I understand if it's not supported.

**Comment by tmcconechy** [comment]
Its because of the way the homepage.js works. It finds all of the cards in the page and tries to lay them out in a way that fits (bin packing algorithm). So the extra card is effecting things. I think if you just want another level of buttons a toolbar or that right floated button or the group action menu underneath will do. Rather than adjusting the algorithm. Hope you can add something in home pages to support `toolbar` or `card-group-action` instead.

Basically a card is just the white background thing, so i dont think it makes sense to nest them. So ii'll close this for now.
